docs/internal/github_actions.rst

```diff
@@ -248,25 +247,25 @@ Each test has a list of the builds it was failing on before being blocked.
 
 * tests/DCPS/Instances/run_test.pl [Multiple Configurations]
 
-  * ``u18_bsafe_js0_FM-1f``
+  * ``test_u18_bsafe_js0_FM-1f``
 
-  * ``u18_esafe_js0``
+  * ``test_u18_esafe_js0``
 
 * tests/DCPS/MultiDPTest/run_test.pl
 
-  * ``u18_bsafe_js0_FM-1f``
+  * ``test_u18_bsafe_js0_FM-1f``
 
-  * ``u18_esafe_js0``
+  * ``test_u18_esafe_js0``
 
 * tests/DCPS/NotifyTest/run_test.pl
 
 * tests/DCPS/Reconnect/run_test.pl restart_pub
 
-  * ``test_w16_x86_i0_sec``
+  * ``test_w22_x86_i0_sec``
 
 * tests/DCPS/Reconnect/run_test.pl restart_sub
 
-  * ``test_w16_x86_i0_sec``
+  * ``test_w22_x86_i0_sec``
 
```

Comment:
Remove from .rst now that blocker is gone

Reply:
These tests still failed (https://github.com/objectcomputing/OpenDDS/runs/5239775457?check_suite_focus=true) so I added them back to the list of blocked tests.